snuba/processor.py

```diff
@@ -101,6 +101,8 @@ class ReplacementType(str, Enum):
         ReplacementType.TOMBSTONE_EVENTS,
         ReplacementType.EXCLUDE_GROUPS,
         ReplacementType.REPLACE_GROUP,
+        ReplacementType.START_UNMERGE_HIERARCHICAL,
```

Comment:
those message types definitely exist in Sentry, but the product logic is behind a featureflag nobody has. I suppose we would be crashing the consumer today if they aren't here.